canonical-web-and-design/jaas.ai
# Add static hashing

## Done

- Added hashes to static URLs. Mostly the same as: https://github.com/canonical-websites/snapcraft.io/blob/83fe601711ac2437b70afa4d7b4dca7807183fd3/webapp/template_utils.py#L58

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Inspect some static files and check that they have an appended hash.

## Details

- Fixes: #164.
